tensorrt_llm/_torch/auto_deploy/custom_ops/flashinfer_rope.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple
+
+import flashinfer
+import torch
+
+
+@torch.library.custom_op("rope::flashinfer", mutates_args=())
+def apply_rope_with_input_pos_flashinfer(
+    q: torch.Tensor, k: torch.Tensor, cos: torch.Tensor, sin: torch.Tensor, is_neox: bool
```

Comment:
```suggestion
    q: torch.Tensor, k: torch.Tensor, cos: torch.Tensor, sin: torch.Tensor, is_neox: bool = True
```
To keep it consistent with the `flashinfer` usage of this parameter